frontend/src/stores/media/index.ts

```diff
@@ -442,9 +443,8 @@ export const useMediaStore = defineStore("media", {
       mediaType: SupportedMediaType
       shouldPersistMedia: boolean
     }) {
-      const queryParams = prepareSearchQueryParams({
-        ...useSearchStore().searchQueryParams,
-      })
+      const queryParams = useSearchStore()
+        .searchQueryParams as PaginatedSearchQuery
```

Comment:
Why is the case necessary here? Could we move it into the search store's `searchQueryParams` getter? Is there some other way to refine this type that wouldn't require a cast?

Reply:
`search` store can compute all of the parameters except for the `page`, which is saved in the `media` store.
I was trying not adding the `page` parameter to the `queryParams` when sending the first search request, that's why I wrote this code like this.
I refactored it now to remove the type cast.